src/components/ButtonDropdown/Toggle.jsx

```diff
@@ -0,0 +1,26 @@
+import { createElement as E } from 'react'
+import T from 'prop-types'
+import { classNames } from '../../utils/'
+
+const initial = 'c-btn-dropdown__toggle'
```

Comment:
don't really need this in a variable, it's only used once so just put the string as a literal in `classNames`

Reply:
done